lib/console_web/schema/schema.ex

```diff
@@ -124,14 +111,21 @@ defmodule ConsoleWeb.Schema do
       arg :user_id, :string
       resolve(&Console.Devices.AuditResolver.paginate/2)
     end
+
+    @desc "Get recent events for a context (packet graph)"
+    field :recent_events, list_of(:event) do
+      arg :context_id, :string
+      arg :context_name, :string
+      resolve &Console.Events.EventResolver.recent/2
+    end
   end
 
   subscription do
     field :event_added, :event do
-      arg :device_id, :string
+      arg :context_id, :string
 
       config fn args, _ ->
-        {:ok, topic: args.device_id}
+        {:ok, topic: args.context_id}
```

Comment:
yeah the subscription is just taking a context_id, so maybe we should make the topic like `"#{args.context_name}/#{args.context_id}"`